### PR TITLE
fix light on/off state value template to use lowercase values

### LINKED ISSUE #288: [bug] All light devices showing error in HASS: Invalid state value received
**Version**

Build/Run method
- [x] Docker
- [ ] PKG
- [ ] Manually built (git clone - npm install - npm run build ) 

Zwave2Mqtt version: master branch
Openzwave Version: 1.6 - master branch

**Describe the bug**
All light devices are throwing this error in HASS:
```
hass    | 2020-03-03 06:17:21 WARNING (MainThread) [homeassistant.components.mqtt.light.schema_template] Invalid state value received
```
All of the lights are now `unknown` in HASS.
The discovery payload looks like this:
```
{
  "type": "light",
  "object_id": "dimmer",
  "discovery_payload": {
    "schema": "template",
    "brightness_template": "{{ (value_json.value / 99 * 255) | round(0) }}",
    "state_topic": "z2m/30/38/1/0",
    "state_template": "{{ \"OFF\" if value_json.value == 0 else \"ON\" }}",
    "command_topic": "z2m/30/38/1/0/set",
    "command_on_template": "{{ ((brightness / 255 * 99) | round(0)) if brightness is defined else 255 }}",
    "command_off_template": "0",
    "device": {
      "identifiers": [
        "zwave2mqtt_0xc5b5c9e5"
      ],
      "manufacturer": "Linear (Nortek Security Control LLC)",
      "model": "WD500Z-1 Wall Dimmer Switch (0x3034)",
      "name": "dining_lights",
      "sw_version": "5.41"
    },
    "name": "dining_lights_dimmer",
    "unique_id": "zwave2mqtt_0xc5b5c9e5_30-38-1-0"
  },
  "discoveryTopic": "light/dining_lights/dimmer/config",
  "values": [
    "38-1-0"
  ],
  "persistent": false,
  "id": "light_dimmer"
}
```
I believe this change broke it, form PR #270: https://github.com/OpenZWave/Zwave2Mqtt/commit/2314898f4cc57debd15e74cb60cbd0b98001d226#diff-42b2a4d55075f77590c8aa76053c2f82L180
Here's the reason I believe that caused the error:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/mqtt/light/schema_template.py#L223-L228
https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L200-L201
The states need to be lowercase `on` and `off`. Not sure what the reasoning was to set those to uppercase in that change.

When I manually change the `HASS Device JSON` to use `on` `off` instead of `ON` `OFF` things work properly again.

**To Reproduce**

**Expected behavior**
The correct value should be extracted.

**Additional context**

### PR DESCRIPTION
**Pull request by chilicheech**
this fixes #288

### CODE REVIEW

**Comment by robertsLando** [comment]
@chilicheech This cannot be merged 

https://www.home-assistant.io/integrations/light.mqtt/#payload_off

https://www.home-assistant.io/integrations/light.mqtt/#payload_on

Update to latest hass version

File: hass/configurations.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ module.exports = {
       brightness_state_topic: true,
       brightness_command_topic: true,
       on_command_type: 'first',
-      state_value_template: '{{ "OFF" if value_json.value == 0 else "ON" }}',
+      state_value_template: '{{ "off" if value_json.value == 0 else "on" }}',
```
Comment (robertsLando) [inline hass/configurations.js]: I think that this should be left to `ON` and `OFF` as our discussion. Just lights with `template` schema needs to use `on|off` lower cases

Comment (chilicheech) [reply hass/configurations.js]: fixed.. keeping this with `ON` and `OFF` and changed just the light dimmer to `on` and `off`

**Comment by robertsLando** [comment]
LGTM. Thanks @chilicheech

**Pull request merged**